NickvisionMoney.Shared/Controllers/AccountViewController.cs

```diff
@@ -410,19 +410,31 @@ private int SortTransactions(uint a, uint b)
         }
         else if (SortTransactionsBy == SortBy.Date)
         {
-            compareTo = _account.Transactions[a].Date.CompareTo(_account.Transactions[b].Date);
+            compareTo = GetCompareToByDate();
         }
         else if (SortTransactionsBy == SortBy.Amount)
         {
-            var aAmount = _account.Transactions[a].Amount * (_account.Transactions[a].Type == TransactionType.Income ? 1m : -1m);
-            var bAmount = _account.Transactions[b].Amount * (_account.Transactions[b].Type == TransactionType.Income ? 1m : -1m);
-            compareTo = aAmount.CompareTo(bAmount);
+            compareTo = GetCompareToByAmount();
```

Comment:
Same here make it:
```csharp
var aAmount = _account.Transactions[a].Amount * (_account.Transactions[a].Type == TransactionType.Income ? 1m : -1m);
var bAmount = _account.Transactions[b].Amount * (_account.Transactions[b].Type == TransactionType.Income ? 1m : -1m);
compareTo = aAmount.CompareTo(bAmount);
if(compareTo == 0)
{
    compareTo = _account.Transactions[a].Date.CompareTo(_account.Transactions[b].Date);
    compareTo = compareTo == 0 ? a.CompareTo(b) : compareTo;
}
```

No need to extract to a method for the one 2 extra lines expanding out the code would add...